chart/templates/webserver/webserver-service.yaml

```
@@ -51,3 +51,9 @@ spec:
   {{- if .Values.webserver.service.loadBalancerIP }}
   loadBalancerIP: {{ .Values.webserver.service.loadBalancerIP }}
   {{- end }}
+  {{- if .Values.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:
+  {{- range $cidr := .Values.service.loadBalancerSourceRanges }}
+    - {{ $cidr }}
+  {{- end }}
```

```suggestion
  {{- toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
```

I think this is cleaner?